pkg/retry/retry.go

```diff
@@ -16,6 +16,6 @@ package retry
 import "time"
 
 const (
-	DefaultLinearBackoffInterval = time.Second
+	DefaultLinearBackoffInterval = time.Minute
```

Comment:
I am more worried that the retry interval is so long, resulting in high CPU usage.

Reply:
agree, I guess we should use time.Minute for TLS setup only instead of changing the default value.

Reply:
@beiwei30 Are you suggesting to have a separate backoffInterval for mtls? Currently this retry is being used in this file [direct_messaging.go](https://github.com/dapr/dapr/blob/master/pkg/messaging/direct_messaging.go), which seems to be independent of mTLS setting. Which files do you suggest I should take a look at?

Reply:
You should use a separate instance of `dapr/kit` retries just for service invocation retries.

Please do not change a default that is used elsewhere - that is not the correct approach.

https://github.com/dapr/kit/blob/main/retry/retry.go

Reply:
@pravinpushkar, correct, we shouldn't change this default value. Bernd made a very good point, pls. follow his suggestion.